sdlib/national_districts.py

```diff
@@ -17,32 +10,446 @@ def populate(cur, reset=False):
           cur: a database cursor
           reset: whether to empty the table before populating it (warning!)
         """
-        with urllib.request.urlopen(LEGISLATORS_URL) as l:
-            legislators = yaml.load(l)
-
-        state_districts = defaultdict(list)
-        for rep in legislators:
-            term = rep['terms'][-1]
-            if term['type'] == 'rep':
-                state_districts[term['state']].append(term['district'])
-
-        states = list(state_districts.keys())
-        states.sort()
-        ndtuples = []
-        for state in states:
-            districts = state_districts[state]
-            districts.sort()
-            for district in districts:
-                # Don't let MySQL assign AUTO_INCREMENT ids to the
-                # rows, because we want them to start at 1
-                id = 1 + len(ndtuples)
-                district_abbr = '%s-%s' % (state, district)
-                ndtuples.append((id, state, district, district_abbr))
-
         if reset:
             cur.execute('DELETE FROM national_districts')
 
         sql = ('INSERT INTO national_districts ' +
```

Comment:
we probably want a comment somewhere of how we are treating "at-large" districts?